src/database/migrations/20200312122638-create-request.js

```diff
@@ -49,6 +49,14 @@ module.exports = {
       allowNull: false,
       type: Sequelize.BOOLEAN,
     },
+    handledBy: {
+      allowNull: true,
+      type: Sequelize.INTEGER,
+      references: {
+        model: 'users',
+        key: 'id',
+      },
+    },
```

Comment:
You cannot edit old migrations. Create new ones as we talked about during the call